ac.js

```diff
@@ -395,7 +397,7 @@ AC.prototype.click = function click(e) {
 
   if (rowid > -1) {
     this.selectedIndex = rowid;
-    this.trigger(rowid);
```

Comment:
I should note that `rowid` is not in fact actually used by `trigger()`, which just accesses the selectedIndex directly, so it seemed fine to simply replace the argument here.

Reply:
That's right - thanks for noticing this!